woocommerce/woocommerce-android
# Issue/4947 Copy to clipboard a readers name

Closes: #4783 
Closes: #4947


### Description


The PR:
* Adds copy in clipboard action on the long click on Reader's name
* Removes cutting right part of the firmware version's name
* Extracts copyToClipboard to an extension function a uses it in respective places

### Testing instructions

* Connect to a reader
* Notice that firmware version now is fully visible
* Long click on a reader's name
* Notice that you have the name in your clipboard

### Images/gif




https://user-images.githubusercontent.com/4923871/136180962-a45a5ddb-d289-46c2-8af1-7ca07b2a5010.mp4
